---
title: Meta-analysis of fecal metagenomes reveals global microbial signatures that
  are specific for colorectal cancer
authors:
- Jakob Wirbel
- Paul Theodor Pyl
- Ece Kartal
- Konrad Zych
- Alireza Kashani
- Alessio Milanese
- Jonas S. Fleck
- Anita Y. Voigt
- Albert Palleja
- Ruby Ponnudurai
- Shinichi Sunagawa
- Luis Pedro Coelho
- Petra Schrotz-King
- Emily Vogtmann
- Nina Habermann
- Emma Nimeus
- Andrew M. Thomas
- Paolo Manghi
- Sara Gandini
- Davide Serrano
- Sayaka Mizutani
- Hirotsugu Shiroma
- Satoshi Shiba
- Tatsuhiro Shibata
- Shinichi Yachida
- Takuji Yamada
- Levi Waldron
- Alessio Naccarati
- Nicola Segata
- Rashmi Sinha
- Cornelia M. Ulrich
- Hermann Brenner
- Manimozhiyan Arumugam
- Peer Bork
- Georg Zeller
short_description: 'Through extensive validations, this meta-analysis firmly establishes globally generalizable, predictive taxonomic and functional microbiome CRC signatures as a basis for future diagnostics.'
journal: Nature Medicine
doi: 10.1038/s41591-019-0406-6
year: 2019
date: 2019-4-1
---

Association studies have linked microbiome alterations with many human diseases. However, they have not always reported consistent results, thereby necessitating cross-study comparisons. Here, a meta-analysis of eight geographically and technically diverse fecal shotgun metagenomic studies of colorectal cancer (CRC, n = 768), which was controlled for several confounders, identified a core set of 29 species significantly enriched in CRC metagenomes (false discovery rate (FDR) < 1 × 10−5). CRC signatures derived from single studies maintained their accuracy in other studies. By training on multiple studies, we improved detection accuracy and disease specificity for CRC. Functional analysis of CRC metagenomes revealed enriched protein and mucin catabolism genes and depleted carbohydrate degradation genes. Moreover, we inferred elevated production of secondary bile acids from CRC metagenomes, suggesting a metabolic link between cancer-associated gut microbes and a fat- and meat-rich diet. Through extensive validations, this meta-analysis firmly establishes globally generalizable, predictive taxonomic and functional microbiome CRC signatures as a basis for future diagnostics.

